### DISCUSSION TITLE
Is word level timestamp supported?

### DISCUSSION THREAD

**Discussion by BadarSaghir** [SYSTRAN/faster-whisper#290] [Q&A] [answered]
I have seen  linto-ai/whisper-timestamped project support the word level timestamp in implementation. Is their any parameter that can give the word level timestamp for segments?

https://github.com/linto-ai/whisper-timestamped

**Comment by hoonlight** [comment, marked as answer]
Try `word_timestamps=True`

**Reply by BadarSaghir** [reply]
Thanks
